testing/js/src/test/scala/org/http4s/Http4sSuitePlatform.scala

```diff
@@ -24,5 +24,3 @@ trait Http4sSuitePlatform { this: Http4sSuite =>
   override def munitFlakyOK =
     Try(js.Dynamic.global.process.env.CI).toOption.filterNot(js.isUndefined).isDefined
 }
```

Comment:
Just realized, once `Env` gets released in CE 3.4 we'll be able to deplatform this :)